### DISCUSSION TITLE
Organização do projeto

### DISCUSSION THREAD

**Discussion by ortense** [mist-gopher/feature-tag#2] [Request for Comments]
![image](https://github.com/mist-gopher/feature-tag/assets/1520183/d50ec4d8-42f0-4d48-8331-6c2c5ca4ab98)

Como o intuito é estudo acho que podemos utilizar o  a estrutura sugerida pelo [golang-standards/project-layout](https://github.com/golang-standards/project-layout) para organizarmos o projeto.

Particularmente tem algumas coisas que eu não gosto, por exemplo o `Dockerfile` não estar na raiz, mas vale a pena tentarmos e adaptar caso algo não atenda a nossa necessidade.

O que acham? 🤔

**Comment by tassiorego** [comment]
Dei uma olhada rápida e acho uma boa opção.
